### ISSUE TITLE
Data Preprocessing

### ISSUE THREAD

**Issue by Temiloluwa** [Temiloluwa/leo_segmentation#4]
- [x] Determine format of input data

- [x] Create Data loaders

- [x]  Create Config file for input parameters

- [x] Code  Data Preprocessing

- [x] Determine structure of masks

- Mask will be binary images (0,1)
- When data is sampled for each task, multiply masks by an integer from 1 to num_tasks.
- The mask can be used as it is if the loss function can perform cross entropy loss with sparse inputs. 

- Else write function to convert a mask to such that its depth dimension are size (num_tasks) and pixels are one-hot-encoded along the depth dimension.
